### ISSUE TITLE
CSS examples: example needed for width

### ISSUE THREAD

**Issue by wbamberg** [mdn/interactive-examples#565]
https://developer.mozilla.org/en-US/docs/Web/CSS/width

The page already contains an example that we should be able to interactivise:

<img width="1053" alt="screen shot 2018-02-08 at 10 21 47 pm" src="https://user-images.githubusercontent.com/432915/36014409-7caa72ce-0d1e-11e8-977e-1fef095ac23a.png">

We should omit any values that don't have good cross-browser support though.

**Comment by dominicduffin1** [comment]
Am I correct in thinking that the interactivised example should only include the blue bits?

**Comment by wbamberg** [comment]
Yes, kind of. I think the output pane should contain one blue block then each choice in the left-hand pane should be the contents of one of the blocks in that example, like:

```
width: 150px;
```

```
width: 20em;
```

```
 width: 75%;
```

```
...and so on.
```

At least, that seems like a reasonable approach for this one :).

**Comment by dominicduffin1** [comment]
That sounds reasonable :) I'll have a go at that.
